rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -27,17 +44,15 @@ trait LocationScaleFamily { self =>
   def logDensity(x: Real): Real
   def generate(r: RNG): Double
 
-  val standard: Continuous = new Continuous {
+  val standard: StandardContinuous = new StandardContinuous {
```

Comment:
the type annotation here should just be `Continuous` to keep `StandardContinuous` private